recipe/meta.yaml

```diff
@@ -24,19 +21,35 @@ requirements:
 
 test:
   commands:
+    # binaries
     - mdb_copy -V                                    # [unix]
     - mdb_dump -V                                    # [unix]
     - mdb_load -V                                    # [unix]
     - mdb_stat -V                                    # [unix]
-    - test -f ${PREFIX}/lib/liblmdb.a                # [unix]
+
+    # shared:
     - test -f ${PREFIX}/lib/liblmdb.so               # [unix]
+    - if not exist %LIBRARY_BIN%\lmdb.dll exit 1     # [win]
```

Comment:
Should we test for the corresponding `.lib` as well?

Reply:
I don't claim to understand all of the windows specifics, but it is my understanding that not every DLL _needs_ a `.lib`? I'd cross-check the logs, but then, we're not building windows yet. I plan to march through the existing [tags](https://github.com/LMDB/lmdb/tags) before attempting that.

Reply:
IIUC we do need it for [dynamic linking on Windows]( https://docs.microsoft.com/en-us/troubleshoot/windows-client/deployment/dynamic-link-library#load-time-dynamic-linking )

That said, given we are not attempting building on Windows yet, maybe we can leave out the Windows changes until we intend to add it

Reply:
I don't see the issue with two extra lines of tests, but if you say so...